### ISSUE TITLE
FR: Labels inline with Highlight

### ISSUE THREAD

**Issue by profucius** [omnivore-app/obsidian-omnivore#22]
Great plugin and service! I'm considering switching from Readwise, currently testing the featureset of your app. If this feature was already mentioned somewhere or is included and I missed it, then I apologize in advance.

I would like for my highlights to be tagged inline with the label I assigned inside Omnivore. So, I highlight a line, assign that highlight a label, then set my Template (within Obsidian plugin) to include something like `{{highlightLabels}}` at the end of the line for highlights. After syncing to Obsidian, the result would be the file with the highlighted line(s), each with any label(s) assigned to the highlight formatted as a `#tag` (in my case, at the end of the line).

The benefit of this would be that searching inside of Obsidian for a tag would provide results of the line(s) that were highlighted. Instead of just the article itself, it would be possible to find specific highlights by a searched tag.

Again, if this is already possible then I missed that mentioned in your documentation. I tried `{{highlightLabels}}` but that did not work.

**Comment by sywhb** [comment]
Working in #38

**Comment by sywhb** [comment]
Hey @profucius, we have released a new version 1.1.1 to address this issue.

Now you can use the `{{labels}}` variable which has a `{{name}}` attribute, in your template.

I have tagged my highlight with `bug` and `test` labels:
<img width="664" alt="Screenshot 2023-03-31 at 11 29 58 AM" src="https://user-images.githubusercontent.com/12963965/229017271-98eb19a3-c8a8-40d9-9567-c29fd5fcecce.png">

The highlight section in the template:
```
> {{{text}}} [⤴️]({{{highlightUrl}}}) {{#labels}} #{{name}} {{/labels}}
```
